packages/ui/src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -252,6 +375,13 @@ export default {
         selectedObject: undefined,
         withDefault: undefined,
       },
+      baseSwitchData: {
+        options: ['test switch', 'test switch 1', 'test switch 2'],
+        // bug when selected is true
```

Comment:
It would make sense to create a separate card and mark it in some way to indicate it's not intended behaviour. Even better is to fix it.